vector/src/main/java/im/vector/app/features/contactsbook/ContactsBookFragment.kt

```diff
@@ -67,13 +68,17 @@ class ContactsBookFragment @Inject constructor(
         setupFilterView()
         setupConsentView()
         setupOnlyBoundContactsView()
-        setupCloseView()
         contactsBookViewModel.observeViewEvents {
             when (it) {
                 is ContactsBookViewEvents.Failure             -> showFailure(it.throwable)
                 is ContactsBookViewEvents.OnPoliciesRetrieved -> showConsentDialog(it)
             }.exhaustive
         }
+        configureToolbar(views.phoneBookToolbar)
+                .withTitle(R.string.contacts_book_title)
+                .allowBack(true)
+                .withCustomBackIcon(R.drawable.ic_x_18dp)
```

Comment:
I'm not sure if we use X button everyevere, if so, it could be better to make it default and call this method only for other cases